### PR TITLE
Allow Active Record 5.2 to be used

### PR DESCRIPTION
**Pull request by pixeltrix**
Also bump the Ruby versions tested against.

### CODE REVIEW

**Comment by pixeltrix** [comment]
Apart from the rubocop failure it looks like it's failing to install the delayed_job gem because the released version still depends on < 5.2. Hopefully when/if collectiveidea/delayed_job#1023 is merged and released then they'll pass.

File: gemfiles/postgresql/5-2.gemfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  platforms :ruby, :mswin, :mingw do
+    gem 'pg', "~> 0.18"
+  end
+
+  gem 'coveralls', :require => false
+  gem 'rspec', '>= 2.11'
+  gem 'simplecov', :require => false
+
+  gem 'activerecord', "~> 5.1.0"
```
Comment (y-takey) [inline gemfiles/postgresql/5-2.gemfile]: Is correct this version? `~> 5.2.0`?

Comment (pixeltrix) [reply gemfiles/postgresql/5-2.gemfile]: Argh so many numbers - will fix

**Comment by FanaHOVA** [comment]
Is this working for you? It looks like `delayed_job` still makes the build fail. `5.2.rc2` works fine though.

```
Bundler could not find compatible versions for gem "activesupport":
  In Gemfile:
    delayed_job_active_record was resolved to 4.1.2, which depends on
      delayed_job (< 5, >= 3.0) was resolved to 3.0.5, which depends on
        activesupport (~> 3.0)

    rails (= 5.2.0) was resolved to 5.2.0, which depends on
      activesupport (= 5.2.0)
```

**Comment by wnm** [comment]
@FanaHOVA [this pr](https://github.com/collectiveidea/delayed_job/pull/1023) in delayed_job must be merged before. currently delayed_job depends on activesupport ~> 3.0

**Comment by pixeltrix** [comment]
@wnm technically it's `>= 3.0, < 5.2` which means that it works with 5.2 RC gems but not the final version. 🤦‍♂️

**Comment by GuillaumeWrobel** [comment]
`DelayJob` pull request is merged into master! Lets update to Rails 5.2 :100:

**Comment by albus522** [comment]
Has a couple of issues but I'll take it from here

**Comment by toao** [comment]
@albus522 thanks for merging, looking forward to the release once the remaining issues are resolved 👍

**Comment by oniofchaos** [comment]
Is there a list/issues/PRs for the other items that are needed before this can be bumped on Rubygems?

**Comment by albus522** [comment]
It is out now

**Pull request merged**